### DISCUSSION TITLE
Changelogs

### DISCUSSION THREAD

**Discussion by rogercreagh** [TechbeeAT/jtxBoard#519] [General]
~~This is perhaps a github issue but it would be nice to see a changelog - a simple list of the titles of issues addressed with link to the issue discussions would do - for each release. Everything changed since the previous release.~~

~~It may be there is a way to filter the issues list that would give me what I want, but I can't work it out.~~ 

~~I feel that the release view should automatically generate it,but it doesn't.~~

Deleted 'cos I then instantly found it. Closed issues  |  Milestones

**Comment by patrickunterwegs** [comment]
Hey @rogercreagh , 
I'm creating the releases automatically, but not the release notes. I don't like having all issues in the release notes, so I'm usually writing a summary of the most important changes and add them manually after. Maybe you just looked while the release was already created but I hadn't put the release summary yet!
